modules/ztp-high-security-issues-addressed.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-security-hardening.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-addressed-security-issues_{context}"]
+= Security hardening measures implemented
+
+== Disabling USB boot, wireless and bluetooth access  
+
+To prevent unauthorized modifications to the operating system and to reduce the risk of physical attack vectors, USB boot, wireless, and Bluetooth are disabled in the BIOS settings. These setting are not controlled through ZTP configuration. For example, disabling USB boot prevents attackers from booting from a USB drive and gaining unauthorized access to the system. The recommended firmware configuration is described in "Configuring host firmware for low latency and high performance".
+
+== Preventing unauthorized Reboots
+ 
+Disabling `CtrlAltDelBurstAction` and `Ctrl+Alt+Del` prevents unauthorized reboots.
```

Comment:
Should there be a noun after these? Like "Disabling the `CtrlAltDelBurstAction` and `Ctr+Alt_Del` inputs..." or...?